policies.go

```diff
@@ -263,9 +263,6 @@ type tokenAwareHostPolicy struct {
 }
 
 func (t *tokenAwareHostPolicy) SetPartitioner(partitioner string) {
-	t.mu.Lock()
```

Comment:
im not sure removing this lock is correct, as its protecting concurrent access to `t.partitioner`